internal/utils/templates/init_config.test.toml

```diff
@@ -99,6 +99,8 @@ content_path = "./supabase/templates/invite.html"
 enable_signup = true
 # If enabled, users need to confirm their phone number before signing in.
 enable_confirmations = false
+# Template for sending OTP to users
+template = "Your code is {{ .Code }} ."
```

Comment:
**suggestion:**
As this is parsed as if it was an actual go template, it is failing because it can't find the reference to ".Code".
I would change this to 
```toml
template = "Your code is {{ `{{.Code }}` }} ."
```
So the output in `config.yaml` is 
```toml
template = "Your code is {{ `{{.Code }}` }} ."
```